composer.json

```diff
@@ -22,6 +21,7 @@
         "symfony/yaml": "~2.7|~3.0|~4.0"
     },
     "require-dev": {
+        "composer/composer": "^1.0",
```

Comment:
No need for this dependency anymore right?

Reply:
Not sure. I check this in Symfony Flex. There, they do it too.